nextcloud_install_production.sh

```diff
@@ -824,6 +824,7 @@ download_script GITHUB_REPO nextcloud-startup-script
 download_script STATIC instruction
 download_script STATIC history
 download_script NETWORK static_ip
+download_script NOT_SUPPORTED not-supported
```

Comment:
would it be okay in the install_production script?

Reply:
hehe, no, same thing. :)

Reply:
In this case, I guess this PR doesn't make much sense at all